sdk/textanalytics/Azure.AI.TextAnalytics/tests/RecognizeHealthcareEntitiesTests.cs

```diff
@@ -0,0 +1,230 @@
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(1, resultCollection.Count);
+            Assert.AreEqual(6, resultCollection[0].Entities.Count);
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.AreEqual("rapid", resultCollection[0].Entities.FirstOrDefault().Text);
+            Assert.AreEqual("SymptomOrSign", resultCollection[0].Entities.FirstOrDefault().Category);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchWithErrorTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "Subject is taking 100mg of ibuprofen twice daily",
+                "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure.",
+                "",
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.IsNotNull(resultCollection[0].Id);
+            Assert.IsNotNull(resultCollection[0].TextAnalyticsError);
+            Assert.AreEqual("Document text is empty.", resultCollection[0].TextAnalyticsError.Message);
+            Assert.AreEqual(TextAnalyticsErrorCode.InvalidDocument, resultCollection[0].TextAnalyticsError.ErrorCode.ToString());
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchConvenienceTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.AreEqual(resultCollection.Count(), 2);
+        }
+
+        [Test]
+        public async Task RecognizeHealthcareEntitiesBatchConvenienceWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = batchConvenienceDocuments;
+
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                IncludeStatistics = true
+            };
+
+            HealthcareOperation operation = await client.StartHealthcareBatchAsync(documents, "en", options);
+
+            await operation.WaitForCompletionAsync();
+
+            RecognizeHealthcareEntitiesResultCollection resultCollection = operation.Value;
+
+            Assert.GreaterOrEqual(resultCollection.Count(), 2);
+
+            Assert.Greater(resultCollection.Statistics.DocumentCount, 0);
```

Comment:
you could do `Assert.AreEqual(documents.Length(), resultCollection.Statistics.DocumentCount);`